conflicts.rb

```diff
@@ -25,8 +26,14 @@ def get_changed_files(change_json)
 current_change_json = JSON.parse(change_details)
 current_changed_files = get_changed_files(current_change_json)
 
-open_changes = StringIO.new `ssh -p 29418 gerrit.rfiserve.net gerrit query --format=JSON --files --current-patch-set status:open project:ui/orion`
+project = 'ui/orion'
+if ARGV[0]
+  project = ARGV[0]
```

Comment:
I don't think this works if you're using the other command line args such as --recency. I'd recommend making it a --project= option and then accessing via CMD_LINE_ARGS defined above

Reply:
Updated